site/src/sphinx/server.rst

```diff
@@ -15,3 +15,4 @@ Writing a server
     server-http-file
     server-servlet
     server-access-log
+    server-annotated-service
```

Comment:
We could 1) remove `server-rest` and 2) put `server-annotated-service` after `server-docservice`.